### PR TITLE
Update Postman Collections for Events API

### PR DESCRIPTION
**Pull request by tdraganidis**
Related story AB#126596

**Description**

Adds Events API endpoints to the Postman Collections to facilitate testing and discoverability.

**Implementation**

- [x] Modify Postman collections both for AAD and user credentials.

### CODE REVIEW

File: Postman/Nfield Public API AAD.postman_collection.json [hunk 1]
```diff
@@ -16547,6 +16547,345 @@
...
+									"});\r",
+									"\r",
+									"const validateProperties = (eventSubscription) => {\r",
+									"    return eventSubscription.EventSubscriptionName !== undefined\r",
+									"    && eventSubscription.Endpoint !== undefined\r",
+									"    && eventSubscription.EventTypes !== undefined;\r",
+									"}\r",
+									"\r",
+									"pm.test(\"Validate properties and set Event Subscription Name\", function () {\r",
+									"    const office = pm.response.json();\r",
```
Comment (PhilipLipnitskiNipo) [inline Postman/Nfield Public API AAD.postman_collection.json:16701]: Why this line is needed?

File: Postman/Nfield Public API.postman_collection.json [hunk 2]
```diff
@@ -13238,6 +13239,299 @@
...
+									"});\r",
+									"\r",
+									"const validateProperties = (eventSubscription) => {\r",
+									"    return eventSubscription.EventSubscriptionName !== undefined\r",
+									"    && eventSubscription.Endpoint !== undefined\r",
+									"    && eventSubscription.EventTypes !== undefined;\r",
+									"}\r",
+									"\r",
+									"pm.test(\"Validate properties and set Event Subscription Name\", function () {\r",
+									"    const office = pm.response.json();\r",
```
Comment (PhilipLipnitskiNipo) [inline Postman/Nfield Public API.postman_collection.json:13376]: Same

File: Postman/Nfield Public API.postman_collection.json [hunk 3]
```diff
@@ -13238,6 +13239,299 @@
...
+								":SubscriptionName"
+							],
+							"variable": [
+								{
+									"key": "SubscriptionName",
+									"value": "{{EventSubscriptionName}}"
+								}
+							]
+						},
+						"description": "Update a fieldwork office."
```
Comment (PhilipLipnitskiNipo) [inline Postman/Nfield Public API.postman_collection.json:13530]: It seems that description is wrong

File: Postman/Nfield Public API AAD.postman_collection.json [hunk 4]
```diff
@@ -16547,6 +16547,345 @@
...
+								":SubscriptionName"
+							],
+							"variable": [
+								{
+									"key": "SubscriptionName",
+									"value": "{{EventSubscriptionName}}"
+								}
+							]
+						},
+						"description": "Update a fieldwork office."
```
Comment (PhilipLipnitskiNipo) [inline Postman/Nfield Public API AAD.postman_collection.json:16884]: It seems that description is wrong

File: Postman/Nfield Public API.postman_collection.json [hunk 5]
```diff
@@ -13238,6 +13239,299 @@
...
+							"host": [
+								"{{origin-public-api}}"
+							],
+							"path": [
+								"v1",
+								"Events",
+								"Subscriptions"
+							]
+						},
+						"description": "Gets a list of repository plans."
```
Comment (PhilipLipnitskiNipo) [inline Postman/Nfield Public API.postman_collection.json:13295]:
```suggestion
						"description": "Gets a list of subscriptions."
```

File: Postman/Nfield Public API.postman_collection.json [hunk 6]
```diff
@@ -13238,6 +13239,299 @@
...
+								":SubscriptionName"
+							],
+							"variable": [
+								{
+									"key": "SubscriptionName",
+									"value": "{{EventSubscriptionName}}"
+								}
+							]
+						},
+						"description": "Gets a list of repository plans."
```
Comment (PhilipLipnitskiNipo) [inline Postman/Nfield Public API.postman_collection.json:13354]:
```suggestion
						"description": "Gets a list of subscriptions."
```

File: Postman/Nfield Public API.postman_collection.json [hunk 7]
```diff
@@ -13238,6 +13239,299 @@
...
+							"host": [
+								"{{origin-public-api}}"
+							],
+							"path": [
+								"v1",
+								"Events",
+								"Subscriptions"
+							]
+						},
+						"description": "Provisions a new data repository."
```
Comment (PhilipLipnitskiNipo) [inline Postman/Nfield Public API.postman_collection.json:13418]:
```suggestion
						"description": "Creates subscription."
```

File: Postman/Nfield Public API.postman_collection.json [hunk 8]
```diff
@@ -13238,6 +13239,299 @@
...
+								":SubscriptionName"
+							],
+							"variable": [
+								{
+									"key": "SubscriptionName",
+									"value": "{{EventSubscriptionName}}"
+								}
+							]
+						},
+						"description": "Provisions a new data repository."
```
Comment (PhilipLipnitskiNipo) [inline Postman/Nfield Public API.postman_collection.json:13477]:
```suggestion
						"description": "Deletes a subscription."
```

File: Postman/Nfield Public API AAD.postman_collection.json [hunk 9]
```diff
@@ -16547,6 +16547,345 @@
...
+							"host": [
+								"{{origin-public-api}}"
+							],
+							"path": [
+								"v1",
+								"Events",
+								"Subscriptions"
+							]
+						},
+						"description": "Gets a list of repository plans."
```
Comment (PhilipLipnitskiNipo) [inline Postman/Nfield Public API AAD.postman_collection.json:16611]:
```suggestion
						"description": "Gets a list of subscriptions."
```

File: Postman/Nfield Public API AAD.postman_collection.json [hunk 10]
```diff
@@ -16547,6 +16547,345 @@
...
+								":SubscriptionName"
+							],
+							"variable": [
+								{
+									"key": "SubscriptionName",
+									"value": "{{EventSubscriptionName}}"
+								}
+							]
+						},
+						"description": "Gets a list of repository plans."
```
Comment (PhilipLipnitskiNipo) [inline Postman/Nfield Public API AAD.postman_collection.json:16679]:
```suggestion
						"description": "Gets a subscription."
```

File: Postman/Nfield Public API AAD.postman_collection.json [hunk 11]
```diff
@@ -16547,6 +16547,345 @@
...
+							"host": [
+								"{{origin-public-api}}"
+							],
+							"path": [
+								"v1",
+								"Events",
+								"Subscriptions"
+							]
+						},
+						"description": "Provisions a new data repository."
```
Comment (PhilipLipnitskiNipo) [inline Postman/Nfield Public API AAD.postman_collection.json:16755]:
```suggestion
						"description": "Creates a new subscription."
```

File: Postman/Nfield Public API AAD.postman_collection.json [hunk 12]
```diff
@@ -16547,6 +16547,345 @@
...
+								":SubscriptionName"
+							],
+							"variable": [
+								{
+									"key": "SubscriptionName",
+									"value": "{{EventSubscriptionName}}"
+								}
+							]
+						},
+						"description": "Provisions a new data repository."
```
Comment (PhilipLipnitskiNipo) [inline Postman/Nfield Public API AAD.postman_collection.json:16820]:
```suggestion
						"description": "Deletes a subscription."
```

**Review by PhilipLipnitskiNipo — APPROVED** [review]
Good! Only descriptions should be updated 🙂

File: Postman/Nfield Public API AAD.postman_collection.json (hunk 1 above)
Comment (tdraganidis) [reply Postman/Nfield Public API AAD.postman_collection.json:16701]: This line shouldn't be there as it has been deleted! I will re-upload the exported docs.

**Review by AlbertoBelver — APPROVED** [review]

**Pull request merged**